research/conditional/utils/argparse.py

```diff
@@ -65,7 +65,13 @@ def introduce_parser_arguments(
     parser.add_argument("--mixed_precision", action="store_true")
     parser.add_argument("--torch_compile", action="store_true")
     parser.add_argument("--loss_checkpoint_chungs", type=int, default=0)
-    parser.add_argument("--data_distributed", action="store_true")
+    parser.add_argument("--ddp_enabled", action="store_true")
+    parser.add_argument("--fsdp_enabled", action="store_true")
+    parser.add_argument(
+        "--fsdp_offload_params",
+        action="store_true",
+        help='This option is NOT for turning on CPUOffloading. It is parameter to CPUOffloading class. "If True, then this offloads gradients to CPU as well, meaning that the optimizer step runs on CPU."',
+    )
```

Comment:
Is the description correct? According to the code, if the property is True, then CPU offloading is enabled, and when this is False, CPU offloading is disabled